shopelectro/management/commands/price.py

```diff
@@ -36,13 +69,13 @@ def run(self, query_set: QuerySet) -> QuerySet:
         return self.FILTERS[self.utm](query_set)
 
 
-# @todo #661:120m Refactor price command.
-#  Create Price class with it's own price processing.
-class Command(BaseCommand):
-    """Generate yml file for a given vendor (YM or price.ru)."""
+class Tree:
```

Comment:
This class doesn't look like `Tree` and it sounds very abstract.

It seems `PriceList`/`Target` is more relevant for it